pkg/compiler/check.go

```diff
@@ -325,10 +325,6 @@ func (comp *compiler) checkAttributeValues() {
 			for _, f := range st.Fields {
 				isOut := hasOutOverlay
 				for _, attr := range f.Attrs {
-					desc := structFieldAttrs[attr.Ident]
```

Comment:
I'm thinking about a scenario when/if we actually start doing some checks on per-field attributes. How do we make sure we don't forget to re-implement this code?

Panic if `desc.CheckConsts != nil`? But then it's easier to just keep the code as is.

Maybe at least some comment to `struct attr_desc` then?

Reply:
I'm generally of the opinion that dead code should just be removed. We tend to all be fairly busy people who overestimate when we'll actually reuse such dead code. And in the meantime it can make things a bit harder for new contributors who have to understand that dead code and work with it.

In this case, it probably doesn't matter much either way. I'm also happy to drop this first commit.

> I'm thinking about a scenario when/if we actually start doing some checks on per-field attributes. How do we make sure we don't forget to re-implement this code?

If we do, I guess we'll add tests. Wouldn't they fail on such missing logic?

Reply:
> Wouldn't they fail on such missing logic?

They would, indeed. 

Okay, let's remove the code, but I think we should still leave some comment in `type attrDesc struct`, so that that person whose tests would fail wouldn't have to waste much time debugging this.

Something like `The function is not invoked for per-field attributes, only for whole structs/unions.`?